### PR TITLE
Modified the nginx/phpfpm/redis plugin in python.d

### PR DESCRIPTION
**Pull request by leolovenet**
1. changed arguments from decode() in base.py to support versions of Python 2.6
2. added pass parameters to redis for AUTH command
3. optimized nginx.chart.py and phpfpm.chart.py submenu name
4. optimized phpfpm.chart.py average display

### CODE REVIEW

**Comment by ktsaou** [comment]
thanks!

@paulfantom this is yours...

File: python.d/phpfpm.chart.py [hunk 1]
```diff
@@ -127,3 +137,6 @@ def _get_data(self):
         if len(data) == 0:
             return None
         return data
+
+    def check(self):
```
Comment (paulfantom) [inline python.d/phpfpm.chart.py]: This is not needed since this Service class is a child of UrlService

**Comment by ktsaou** [comment]
@leolovenet I am sorry, but I made changes to python.d.plugin that brake this.
Could you please fix them to merge it?

**Comment by leolovenet** [comment]
sorry for the delay, The work has been a little busy lately.

**Comment by ktsaou** [comment]
ok, nice.
Is this ready to be merged? Shall I test it a bit?

**Comment by leolovenet** [comment]
yes, I have been completely tested under CentOS 6.8 with python 2.6.

**Comment by ktsaou** [comment]
seems good on arch linux with python 3.5 and python2.7
merging it.

**Comment by ktsaou** [comment]
thanks!

**Comment by ktsaou** [comment]
works nice on gentoo too...

**Comment by leolovenet** [comment]
:)

**Pull request merged**